src/routes/safe/store/actions/createTransaction.ts

```diff
@@ -246,14 +247,18 @@ const createTransaction = ({
         return receipt.transactionHash
       })
   } catch (err) {
+    const errorMsg = err.message
```

Comment:
was this a requirement? what kind of errors do you get there? are they going to fit into a notification?

Reply:
The requirement is to show additional info to the user (i.e showing that Contract data is not enabled when transaction fails). I tested it and fits well into a notification

Reply:
oh, cool